tests/unit/type/list.tcl

```diff
@@ -1915,6 +1915,25 @@ foreach {pop} {BLPOP BLMPOP_LEFT} {
         r ping
     } {PONG}
 
+    test "BLPOP/BLMOVE should increase dirty" {
+        r del lst{t} lst1{t}
+        set rd [redis_deferring_client]
+
+        set dirty [s rdb_changes_since_last_save]
+        $rd blpop lst{t} 0
+        r lpush lst{t} a
+        set dirty2 [s rdb_changes_since_last_save]
```

Comment:
Done

Reply:
> i'll argue that there's no reason to mention the dirty counter fix in the release notes, since the command did increment it, and the bug is that it incremented it once instead of twice, probably not a big issue)

Agree with you.